src/views/Graph.vue

```diff
@@ -622,7 +562,12 @@ export default {
 
       // load graph data and run layout
       const { data: elements } = await updateData()
-      cy = await this.$cytoscape.instance
+      const stylesheet = await updateStyle()
+      cy = await cytoscape({
+        container: document.getElementById('cytoscape'),
+        elements: elements,
+        style: stylesheet.style
+      })
```

Comment:
I think this is the actual fix for the bug? Instead of re-using the `this.$cytoscape` instance, we are now creating a new instance. Is that correct @MartinRyan ?

Reply:
As far as I understand it this is getting a reference to the same cytoscape instance, not creating a new one